src/index.html

```diff
@@ -5,8 +5,312 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="MoyoLogo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#apple" class="list__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#samsung" class="list__link">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#smartphones" class="list__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#pc"
+              class="list__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="list__item">
+            <a href="#gadgets" class="list__link">Гаджеты</a>
+          </li>
+          <li class="list__item">
+            <a href="#tablets" class="list__link">Планшеты</a>
+          </li>
+          <li class="list__item">
+            <a href="#photo" class="list__link">Фото</a>
+          </li>
+          <li class="list__item">
+            <a href="#video" class="list__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <section class="card" data-qa="card">
+          <div class="card__photo"></div>
+          <h2 class="card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p>
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="card__price">
+            <p>Цена:</p>
+            <p>69 999 грн</p>
```

Comment:
It is better to put these tags into different classes. It helps you to avoid nesting.